### DISCUSSION TITLE
How to build a simplified model, which includes platform pitch, surge and aerodynamic thrust?

### DISCUSSION THREAD

**Discussion by Melancho7ee** [OpenFAST/openfast#2065] [Q&A]
Dear @jjonkman ,
I am working on investigating the coupling of platform pitch, surge and aerodynamic thrust. Inspired by your previous work [1], [2], you and your students studied the negative damping of platform pitch [1] and surge [2], respectively. Negative damping is both found in the two DOF, separately.

I am curious about how the two DOFs couple with each other? I tried to build a model, which includes platform pitch, surge and aerodynamic thrust as follows
<img width="984" alt="CleanShot 2024-03-02 at 10 04 05@2x" src="https://github.com/OpenFAST/openfast/assets/116873649/7416eeb9-bd6a-4f8a-bfad-f5ee73b7d3cc">
where x_s is the displacement of surge, and x_p for the pitch. L_T is the tower height. F_a is the aerodynamic thrust. However, when I use this reduced-order model for simulation verification, the simulation diverges directly. 

Based on my experience, I think there is an error in the thrust term on the right side of the equation, but I don’t know what is correct. I'd be grateful if you could give me some advice.

Best regards,
Jiaqi

[1] Jonkman J. Influence of control on the pitch damping of a floating wind turbine[C]//46th AIAA aerospace sciences meeting and exhibit. 2008: 1306.
[2] Matha D. Model development and loads analysis of an offshore wind turbine on a tension leg platform with a comparison to other floating turbine concepts: April 2009[R]. National Renewable Energy Lab.(NREL), Golden, CO (United States), 2010.

**Comment by Melancho7ee** [comment]
Besides, I do not where to obtain the CM in z direction, that is, zg, of the whole floating offshore wind turbine, such as NREL 5 MW with ITI Barge.

**Reply by jjonkman** [reply]
Dear @Melancho7ee,

Your linearized equation of motion looks OK to me, assuming (C^HS)_55 is the total hydrostatic restoring in pitch, including the effect of body weight (which also includes m*z_g).  What is your concern about the thrust term?

I don't know the vertical center of mass of the NREL 5-MW baseline wind turbine atop the ITI Energy barge off the top of my head, but you should be able to calculate that by the known masses and center of masses of each component.  The mass, center of mass, and inertia of the rotor-nacelle-assembly of the NREL 5-MW baseline wind turbine is provided in the following topic on our forum: aggregate-mass-in-adams-to-check-nrel-cs-monopile-bmi/696.

Best regards,

**Reply by Melancho7ee** [reply]
Dear @jjonkman ,

Thanks a lot! I have figured it out myself. The previous error was due to me selecting the wrong zg. For NREL 5MW ITIBarge, zg should be near the still water sea level. However, I previously misunderstood it as the height of the center of mass of the 5MW turbine. I wrongly used zg = 64 m and the error then occurred.

After I modified the zg, which can be approximated using zg = 0 m, everything seems okay. Thanks again for your help!

I have no other questions to discuss, and I think this discussion can be closed.

Best regards,
Jiaqi
